Header header

#Angle (0.0-359.999, 0 is forward)
float64 angle

#Thrust (0.0-1.0)
float64 thrust

#Diff (-180.0- 180.0, 0 is no difference between current and yaw)
float64 diff
